features/contract-state/GeneralInfo.tsx

```diff
@@ -76,6 +84,17 @@ const GeneralInfo = () => {
         {priceId ? ethers.utils.parseBytes32String(priceId) : "N/A"}
       </Status>
 
+      <Status>
+        <Label>
+          Estimated Identifier Price (
+          <Link href={sourceUrl} target="_blank" rel="noopener noreferrer">
+            Coinbase Pro
```

Comment:
If we're pulling the price and the source URL from the PriceFeed container, should we also pull the name from there too?

Reply:
Actually we shouldn't really be setting the EMP's identifier in two places since its already being set in the EMPState container which reads from the on-chain contract. I'm going to remove the `identifier` param from the `PriceFeedParams` mapping.

On a separate note, would you prefer to display the identifier's name here in this block? (e.g. "Estimated <IDENTIFIER> Price)